lib/logger.js

```diff
@@ -638,6 +645,12 @@ class Logger extends EventEmitter {
     , httpAgent: undefined
     }
 
+    // Setting this to `false` can avoid the browser console error of:
+    // Refused to set unsafe header "user-agent"
+    if (this.sendUserAgent) {
```

Comment:
@darinspivey, just wondering, how is our tracking gonna be if the clients disable this? Doesn't it mean we lose tracking those clients?

Reply:
People should only disable it if they're seeing the errors.  In that case, we're just getting out of the browser's way since it won't allow the `user-agent` header anyway.  If someone disables this when the error is not a problem, then the agent header will not be sent, but I don't think that's a situation we can control or should worry about.  We don't want to be doing `window` checks in this code, IMO.

Reply:
@darinspivey, this also indicates that we may need to develop another standalone and separate solution for `client`/`browser`-side logging in the future

Reply:
Yes, @mdeltito and I spoke about that recently.  It's looking more and more like the thing we should do.  We're on the same page 👍